test/tests/pbs_smoketest.py

```diff
@@ -1505,3 +1505,31 @@ def test_man_pages(self):
             msg = "Successfully retrieved man page for"
             msg += " %s command" % pbs_cmd
             self.logger.info(msg)
+
+    @tags('smoke')
```

Comment:
There is a class level smoke tag for this class. You do not need to tag this test as smoke explicitly.